web/docs/DebuggingFbgServer.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta } from '@storybook/addon-docs/blocks'; 
+
+<Meta title="Documentation/Debugging Backend (fbg-server)" />
+
+# Setup
+You can manually start only the backend by going to the `fbg-server` folder and using `yarn run start`.
+
+In order to only run the unit tests for the fbg-server, you can use `yarn run jest` in that folder.
+
+# Debugging GraphQL queries
+
+For manually testing the GraphQL queries, you can access the GraphQL console at http://localhost:3001.
```

Comment:
I am pretty sure the GraphQL playground would be at /graphql (i.e., http://localhost:3001/graphql ) -- can you check this?